src/algokit/cli/deploy.py

```diff
@@ -0,0 +1,108 @@
+import logging
+import os
+from pathlib import Path
+
+import click
+
+from algokit.core.constants import DEFAULT_NETWORKS, DEPLOYER_KEY, DISPENSER_KEY, LOCALNET, MAINNET
+from algokit.core.deploy import (
+    execute_deploy_command,
+    get_genesis_network_name,
+    load_deploy_command,
+    load_deploy_config,
+)
+
+logger = logging.getLogger(__name__)
+
+
+def extract_mnemonics(skip_mnemonics_prompts: bool, network: str) -> tuple[str | None, str | None]:  # noqa: FBT001
+    """
+    Extract environment variables, prompt user if needed.
+
+    :param skip_mnemonics_prompts: A boolean indicating if user prompt should be skipped.
+    :param network: The name of the network ('localnet', 'testnet', 'mainnet', etc.)
+    :return: A tuple containing deployer_mnemonic and dispenser_mnemonic.
+    """
+    deployer_mnemonic = os.environ.get(DEPLOYER_KEY, "")
+    dispenser_mnemonic = os.environ.get(DISPENSER_KEY, None)
```

Comment:
why is one of these defaulting to `""` and the other to `None`?
also, you can just use `os.getenv`

Reply:
replaced with getenv, as for "" vs None i believe it was just a typo